README.md

```diff
@@ -1 +1 @@
-# wildlife-api
\ No newline at end of file
+# wildlife-api
```

Comment:
Just curious - what was the change here?

Reply:
There were a lot of small changes triggered by running npm run format. I think we had forgotten to run it since we first added it.